assets/charts/control-plane/calico-host-protection/templates/host-protection.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
 spec:
   failsafeInboundHostPorts: []
   bpfLogLevel: Info
+  wireguardEnabled: true
```

Comment:
@iaguis I think we should make it configurable?

Reply:
@invidian name of variable `enable_cluster_encryption` seems fine?

Reply:
Maybe `encrypt_pods_traffic`? As IMO encrypting cluster could also mean encrypting etcd data, encrypting disks on nodes etc.